### ISSUE TITLE
Issues with installation

### ISSUE THREAD

**Issue by srmnitc** [pyiron/atomistics#217]
The installation of atomistics package in a clean env caused the following issues:

- phonopy is not automatically installed
- structuretoolkit is not installed; and try except block prevents error from being captured

**Comment by jan-janssen** [comment]
Yes, both are optional dependencies of the `atomistics` package: 
https://github.com/pyiron/atomistics/blob/main/pyproject.toml#L41

I guess we should update the error message to advise users to install the optional dependencies. @srmnitc Can you open a pull request?
